docs/en/agent/support-plugins.md

```diff
@@ -13,5 +13,6 @@
 * Database Client
   * `gorm`: [GORM](https://github.com/go-gorm/gorm)
     * [MySQL Driver](https://github.com/go-gorm/mysql)
+  * `mongo`: [Mongo](https://github.com/mongodb/mongo-go-driver)
```

Comment:
Please the whole support-plugins.md to add tested versions.

Reply:
I think we missed that before.

Reply:
updated.